hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/TriggersITest.groovy

```diff
@@ -87,6 +95,216 @@ class TriggersITest extends AbstractITestBase {
     }
 
     @Test
+    void testGroupTrigger() {
+        Trigger groupTrigger = new Trigger("group-trigger", "group-trigger");
+        groupTrigger.setEnabled(false);
+
+        // remove if it exists
+        def resp = client.delete(path: "triggers/groups/group-trigger", query: [keepNonOrphans:false,keepOrphans:false])
+        assert(200 == resp.status || 404 == resp.status)
+
+        // create the group
+        resp = client.post(path: "triggers/groups", body: groupTrigger)
+        assertEquals(200, resp.status)
+
+        resp = client.get(path: "triggers/group-trigger")
+        assertEquals(200, resp.status)
+        groupTrigger = (Trigger)resp.data;
+        assertEquals( true, groupTrigger.isGroup() );
+
+        ThresholdCondition cond1 = new ThresholdCondition("group-trigger", "DataId1-Token",
+            ThresholdCondition.Operator.GT, 10.0);
+        Map<String, Map<String, String>> dataIdMemberMap = new HashMap<>();
+        MemberCondition memberCondition = new MemberCondition( cond1, dataIdMemberMap );
```

Comment:
Some feedback about name of the class.
With MemberCondition my first thought was this class is member of a group. But its meaning is to define the members for a condition. I wonder if ConditionMembers sounds better or if perhaps was my translation (probably it's me, but just in case).